app/src/main/java/org/simple/clinic/forgotpin/confirmpin/ForgotPinConfirmPinScreenController.kt

```diff
@@ -102,7 +101,7 @@ class ForgotPinConfirmPinScreenController @Inject constructor(
         .share()
 
     val makeResetPinCall = validPin
-        .flatMapSingle { newPin -> syncAndClearPatientData().toSingleDefault(newPin) }
+        .flatMapSingle { newPin -> syncAndClearPatientData.run().toSingleDefault(newPin) }
```

Comment:
Curious, why do we have to clear patient data while resetting the user's pin?

Reply:
This is a product decision. We don't know if the user who is attempting to reset the PIN is a legitimate user who has genuinely forgotten their PIN, or someone who is trying to reset the PIN to get access to the data on the phone. To be safe, we clear the patient data stored locally when a user tries to reset their a PIN.

This is expected to be paired with a phone call from the user's supervisor to verify that they are the real user who is trying to reset the PIN before they get access to the patient data again.